shacl/register.ttl

```diff
@@ -173,21 +172,42 @@ reg:SchemaDescriptionProperty a sh:PropertyShape ;
 reg:SchemaDateCreatedProperty a sh:PropertyShape ;
     sh:path schema:dateCreated ;
     sh:minCount 0 ;
-    sh:datatype xsd:date ;
+    sh:or (
+        [
+            sh:datatype xsd:date ;
+        ]
+        [
+            sh:datatype xsd:dateTime ;
+        ]
+    ) ;
```

Comment:
Good change, but missing a test. Trying this locally I find that:

```json
"dateCreated": "2021-05-27"
```

does not work. Is this because Schema.org’s context defines the range as https://schema.org/Date?

Reply:
added xsd:string with pattern to allow for literals

Reply:
`xsd:string` is gone again and now an `xsd:date` (which it should be).